src/components/structures/RoomView.tsx

```diff
@@ -2035,8 +2059,8 @@ export default class RoomView extends React.Component<IProps, IState> {
                         />
                         <MainSplit panel={rightPanel} resizeNotifier={this.props.resizeNotifier}>
                             <div className="mx_RoomView_body">
-                                {auxPanel}
                                 <div className={timelineClasses}>
+                                    {auxPanel}
```

Comment:
this might be problematic with other code like the tray resizing, timeline size compensation, etc - is this needed for the PR?

Reply:
You're right this caused problems. It caused problems even without the change, thereofre I've entirely moved the `fileDropTarget`